### ISSUE TITLE
[feature proposal] Load a model as a "Pattern Library"...

### ISSUE THREAD

**Issue by jbsarrodie** [archimatetool/archi#32]
Hi,

What about adding the ability to load a model as a "Pattern Library"...

This would then open a new tab on Archi, listing all views contained on that model. At first  each view would be collapsed, but with ability to be "opened" to see a small version of the view:

![image](https://user-images.githubusercontent.com/5757396/95580036-f29fc200-0a36-11eb-9eaf-ce1fdf185156.png)

The pattern name would be the view name, and hovering would display the view documentation...

With this "Pattern Library", it could then be really easy to add a whole bunch of elements and relation to a new model (without having to open a model, the view, selecting all, copying, switch to the model being edited, pasting, removing unwanted "(copy)" suffixes...).

So basically its a kind of enhanced copy/past buffer.

Of course, Archi would come with a good pattern file as example ;-)

Regards

JB
